README.md

```diff
@@ -166,9 +166,14 @@ library archives (`.a`).
 build the library binary manually. This can be done with the following command ```sudo apt-get install libgtest-dev && cd /usr/src/gtest && sudo cmake . && sudo make && sudo mv libg* /usr/lib/ ```
 [2] libnorm-dev is needed if your zmq library was built with libnorm, and not needed otherwise
 
-Debian / Ubuntu one liner for all dependencies  
+Install all dependencies at once on Debian/Ubuntu: 
+
 ``` sudo apt update && sudo apt install build-essential cmake pkg-config libboost-all-dev libssl-dev libzmq3-dev libunbound-dev libsodium-dev libunwind8-dev liblzma-dev libreadline6-dev libldns-dev libexpat1-dev doxygen graphviz libpgm-dev```
 
+Install all dependencies at once on OSX:
+
+``` brew upgrade && brew install gcc cmake pkg-config boost zmq libpgm unbound libsodium miniupnpc libunwind-headers xz readline ldns expat doxygen graphviz && brew cleanup ```
```

Comment:
You sure gcc is needed? If you have brew then gcc (which will actually be clang) should already be present.

Reply:
I have not tried compiling without honestly, I just install all dependencies and build from there with no issue. It does not hurt to have it installed IMO. It may be used for other projects.

Reply:
Installing brew's GCC is installing GNU gcc which is not required (and in fact wont even be used by default when building Monero on Mac - clang will be used). 

> I have not tried compiling without honestly

I strongly suggest testing before submitting PRs

Reply:
I have tested on osx. I have compiled with no issues with all these installed. It did not compile without hidapi, so I added and compiled fine. Then made this commit for adding that since it was missing.

Reply:
Have you tested without brew's gcc? A one-liner has to assume the user does not have any of the dependencies and the one-liner should not install anything that is not needed.

Reply:
No I have not tried compiling without GCC, although it does not harm to have it installed. I could remove if it is an issue but I really don't see it as an issue.

Reply:
It is not a dependency so should definitely not be listed as one to install. Particularly gcc which will take a long time to build and install.

Reply:
Also why the `brew upgrade`? _Maybe_ `update` but surely not `upgrade`.

Reply:
brew upgrade is like ``` sudo apt-get update && sudo apt-get upgrade ``` , it updates all brew installed dependencies to latest.

Reply:
We should not be adding in an upgrade. Update fine, but not upgrade, that is for the user to decide when they run.

Reply:
Please edit this PR (however you can) to remove gcc and change the brew upgrade to brew update. Ty

Reply:
Max 50 characters for commit name, should be good to go now.